### DISCUSSION TITLE
Did the ejabberd-XXX folder naming convention change?

### DISCUSSION THREAD

**Discussion by maheshn393** [processone/ejabberd#4196] [General]
It is strange that the ejabberd folder inside the /home/ejabberd/lib directory in Version 24.02 is named ejabberd-0.0.0. In the past, this has been the version number. for e.g. /home/ejabberd/lib/ejabberd-20.12.0. Is this a new approach?

**Comment by prefiks** [comment]
Hello,

No it should still use correct version, looks like 0.0.0 value can only happen when building packet without calling configure script first (or strictly speaking when building without there being vars.config file generated by configure). Could you tell me in from where you got that package that has this issue?

**Reply by maheshn393** [reply]
Hi prefiks - 

I saw this after pulling the ejabberd/ecs:24.02 container image from docker hub.

**Comment by badlop** [comment]
This was in the ecs container image, affected 24.02, was reported in https://github.com/processone/docker-ejabberd/issues/111 , it is solved in git, and container image of the next ejabberd release will use again the previous correct version number.

**Reply by maheshn393** [reply]
Thanks @badlop. Is the next release imminent?. If not, we will continue using 20.12 version. What do you recommend?

**Reply by badlop** [reply]
The next ejabberd release is not planned for imminent. The good news is that, now that the container building is more automatized (and hopefully the automatic code is fixed), the container releases will contain no such problems.

I guess you care about ejabberd library path because you add some new beam files, or modified ejabberd beam files to suit your needs, or need access to the `*.sql` initialization files, or the `captcha*.sh` example scripts, right?

**Reply by maheshn393** [reply]
You are correct, sir. We have a few custom beam files. Will continue with version 20.12 until the next version. Appreciate your attention to this matter.
